package.json

```diff
@@ -71,6 +72,7 @@
     "babel-plugin-parameter-decorator": "1.0.16",
     "babel-plugin-transform-typescript-metadata": "0.3.2",
     "bob-the-bundler": "4.0.0",
+    "cypress": "^11.2.0",
```

Comment:
Why lock the version?

> does it make sens install `cypress` in `packages/web/app` instead of root?

Hmm, good point. I thought it'd be less hassle to have it in the root.

Reply:
The E2Es might grow to more than just the app, and having the tests in the root feels simpler. Lets keep it here for now. 👍